microsoft-edge/extensions-chromium/developer-guide/sidebar.md

```diff
@@ -236,61 +209,65 @@ chrome.tabs.onActivated.addListener(async ({ tabId }) => {
 <!-- ====================================================================== -->
 ## Sidebar extensions user experience
 
-<!-- compare https://developer.chrome.com/docs/extensions/reference/sidePanel/#user-experience -->
-
 Extensions in the Microsoft Edge sidebar have these user experience (UX) features.
 
 
 <!-- ------------------------------ -->
 #### Opening the extension in the sidebar
 
-<!-- compare https://developer.chrome.com/docs/extensions/reference/sidePanel/#open -->
-
-There are several ways to open the extension in the sidebar:
+There are several ways for the user to open the extension in the sidebar:
 
 
 <!-- ---------- -->
-*  Click the **Open in sidebar** icon (![Open in sidebar icon](./sidebar-images/open-in-sidebar-icon.png)), which is displayed next to the extension's name in the Extensions hub:
+###### By clicking an icon
+
+Users can click the **Open in sidebar** icon (![Open in sidebar icon](./sidebar-images/open-in-sidebar-icon.png)), which is displayed next to the extension's name in the Extensions hub:
+
+![Sidebar dialog](./sidebar-images/sidebar-dialog.png)
 
-   ![Sidebar dialog](./sidebar-images/sidebar-dialog.png)
+Or, users can click the extension's custom icon in the toolbar, if it's enabled.  In this example, the extension's custom icon is a circle (![Extension's custom icon](./sidebar-images/custom-icon-for-demo-extension.png)):
```

Comment:
Can we make it clearer here that this user experience works only when the extension has [enabled the shortcut icon to open the sidebar](https://review.learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/developer-guide/sidebar?branch=pr-en-us-2677#enable-the-extensions-shortcut-icon-to-open-the-sidebar)?

Reply:
```
Or, users can click the extension's custom icon in the toolbar, if it's enabled.  
This user experience requires that the extension has enabled the shortcut icon to open the sidebar, 
as described in [Enable the extension's shortcut icon to open the 
sidebar](#enable-the-extensions-shortcut-icon-to-open-the-sidebar) above.  
In this example, the extension's custom icon is a circle 
(![Extension's custom icon](./sidebar-images/custom-icon-for-demo-extension.png)):
```